src/main/java/com/cs56fitnessapp/models/workout/Workout.java

```diff
@@ -67,5 +73,9 @@ public void setTimePerformingHours(double timePerformingHours) {
         this.timePerformingHours = timePerformingHours;
     }
 
+    public double getMET_value(){ return met_Value; }
+
+    public void setMET_value(double met_Value) { this.met_Value = met_Value; }
+
```

Comment:
I have described above why I think we should not have `met_Value` property in the `Workout`.